src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActions/FixAllSuggestedAction.cs

```diff
@@ -62,15 +48,20 @@ public override bool TryGetTelemetryId(out Guid telemetryId)
             return true;
         }
 
-        public string GetDiagnosticID()
-            => Diagnostic.GetTelemetryDiagnosticID();
-
         protected override async Task InnerInvokeAsync(
             IProgressTracker progressTracker, CancellationToken cancellationToken)
         {
             await this.ThreadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
-            using (Logger.LogBlock(FunctionId.CodeFixes_FixAllOccurrencesSession, FixAllLogger.CreateCorrelationLogMessage(FixAllState.CorrelationId), cancellationToken))
+            var fixAllKind = FixAllState.FixAllKind;
+            var functionId = fixAllKind switch
+            {
+                FixAllKind.CodeFix => FunctionId.CodeFixes_FixAllOccurrencesSession,
+                FixAllKind.Refactoring => FunctionId.Refactoring_FixAllOccurrencesSession,
```

Comment:
is there a strong need to have different function ids here?

Reply:
I think we may want to analyze how the FixAll feature is being used for refactorings as opposed to code fixes, and use that to drive prioritizing work to add FixAll support to more of our refactorings.